### DISCUSSION TITLE
How can I create dashboard report in a folder

### DISCUSSION THREAD

**Discussion by radhakrishnanakireddy** [seleniumbase/SeleniumBase#1174] [General]
Hi,
Is there anyway to create dashboard html in a directory instead of root of project.
could you please help me with it

Thank you
RK

**Comment by mdmintz** [comment]
The ``dashboard.html`` file is created in the folder where you kick off ``pytest``. It needs to be created there so that relative links to the log files work properly.

There are a few things you can do to change the location of the dashboard path:

* You could kick off your tests from a different folder, and then the ``dashboard.html`` file will be generated from that folder when you use ``--dashboard``.
* You can use ``--html=report.html`` / ``--html=folder/report.html``, and then the report (which is similar to the dashboard) will be generated in the chosen location.
* You could run a script to move the ``dashboard.html`` file to a new location after your tests complete. You may also want to move the ``./latest_logs/`` folder if you do that.
